shared/src/commonMain/kotlin/io.newm.shared/public/models/User.kt

```diff
@@ -62,3 +62,11 @@ data class User(
     val newPassword: String? = null,
     val confirmPassword: String? = null
 )
+
+fun User.fullName(): String? {
+    return if(!firstName.isNullOrBlank() && !lastName.isNullOrBlank()) "$firstName $lastName" else null
```

Comment:
Some users may have set only one of first name or last name. In the server we use this:
```
"${firstName.orEmpty().trim()} ${lastName.orEmpty().trim()}".trim()
```